samples/display-local-scene/src/main/java/com/esri/arcgismaps/sample/displaylocalscene/screens/DisplayLocalSceneScreen.kt

```diff
@@ -0,0 +1,102 @@
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISScene
+import com.arcgismaps.mapping.ArcGISTiledElevationSource
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.ArcGISSceneLayer
+import com.arcgismaps.mapping.view.Camera
+import com.arcgismaps.mapping.view.SceneViewingMode
+import com.arcgismaps.toolkit.geoviewcompose.LocalSceneView
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun DisplayLocalSceneScreen(sampleName: String) {
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it),
+            ) {
+                val sceneLayer = remember {
+                    ArcGISSceneLayer("https://www.arcgis.com/home/item.html?id=61da8dc1a7bc4eea901c20ffb3f8b7af")
+                }
+
+                val elevationSource = remember {
+                    ArcGISTiledElevationSource("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")
+                }
+
+                val arcGISScene = remember {
+                    ArcGISScene(
+                        BasemapStyle.ArcGISTopographic,
+                        SceneViewingMode.Local
+                    ).apply {
+                        operationalLayers.add(sceneLayer)
+                        baseSurface.elevationSources.add(elevationSource)
+
+                        initialViewpoint = Viewpoint(
+                            center = Point(19455026.8116, -5054995.7415, SpatialReference.webMercator()),
+                            scale = 8314.6991,
+                            camera = Camera(
+                                locationPoint = Point(
+                                    19455578.6821,
+                                    -5056336.2227,
+                                    1699.3366,
+                                    SpatialReference.webMercator()),
+                                heading = 338.7410,
+                                pitch = 40.3763,
+                                roll = 0.0,
+                            )
+                        )
+                        clippingArea = Envelope(
+                            xMin = 19454578.8235,
+                            yMin = -5055381.4798,
+                            xMax = 19455518.8814,
+                            yMax = -5054888.4150,
+                            spatialReference = SpatialReference.webMercator()
+                        )
+                        isClippingEnabled = true
+                    }
+                }
+
+                LocalSceneView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
```

Comment:
Not needed for the Column :) 
```suggestion
                        ,
```

Reply:
Removed but I think that might be coming from the template